src/views/HowToPlay/Layout/Instructions-Layout.tsx

```diff
@@ -15,7 +19,9 @@ export const Home = () => {
         ]}
       ></KeepReading>
 
-      <BackButton />
+      <StyledInstructionsBody>
```

Comment:
I don't think you need this extra component. I think you can just use `<StyledInstructionsLayout />` again if you set the margin property there.

```suggestion
      <StyledInstructionsLayout >
```